pages/_data/equityDashboardUpdateTime.js

```diff
@@ -0,0 +1,20 @@
+// equitydashboardUpdateTime
+//
+// Retrieves json file for use in state dashboard
+//
+const fetch = require('node-fetch')
+
+module.exports = function() {
+  let dataDomain = 'https://raw.githubusercontent.com/cagov/covid-static-data/main/';
```

Comment:
Can we get this from cloudfront or is that possibly not up to date in this case?

Reply:
Fixed.  Was thinking by reading from github during builds, we can slightly speed up rollout, but probably not necessary at this point.

Reply:
Changed back. Let's discuss later -- I'm using github both to speed the pipeline, and so I don't have to do a series of cloudfront updates/cache expiries during the morning update.  I'll work on a longer term fix that does a cloudfront update around 9:55 to insure sync for CDPH.